crates/cli-support/src/js/mod.rs

```diff
@@ -120,8 +119,7 @@ impl<'a> Context<'a> {
             Ok(format!("
                 function(idx) {{
                     // If this object is on the stack promote it to the heap.
-                    if ((idx & 1) === 1)
-                        return addHeapObject(getObject(idx));
+                    if ((idx & 1) === 1) return addHeapObject(getObject(idx));
```

Comment:
I'd prefer we add curly braces to these one-statement blocks but we can bikeshed style and which eslint rules we want to subscribe to in follow up issues.

Reply:
I tried to one-line anything that felt short enough, otherwise I added curly braces. ESLint doesn't actually care about that but the very simple re_indent function I wrote was choking on the second line.